### DISCUSSION TITLE
What is the roadmap

### DISCUSSION THREAD

**Discussion by cedvdb** [invertase/flutterfire_desktop#25] [General]
The official SDKs get updates daily if not hourly by a lot of contributers and googlers. 

So my question is :

 - What's the plan to build a full dart firebase SDK ? Is the plan to hope for community involvement ? 
 - How are we supposed to keep up with the official SDK's ?
 
I guess my question is, what is the roadmap for this to not be a failed experiment ?

Such project won't go well without clear guidelines and goals.

**Comment by pr-Mais** [comment]
Hi!
So the work here started with **core** and **auth**, and they both are so close to be ready.
Also, @TimWhiting have added **functions** which is also going into good progress.

The roadmap for the work on auth, and functions is updated on the [project boards](https://github.com/invertase/flutterfire_desktop/projects?query=is%3Aopen+sort%3Aupdated-desc), you can check it.
